share/spice/seat_geek/events_by_city/seat_geek_events_by_city.css

```diff
@@ -0,0 +1,77 @@
+.zci--seat_geek .tile--seat_geek {
```

Comment:
Oops, didn't realize the scoping class was based off that ID. Makes sense in retrospect. Fixed in latest commit.